bot/utils/channel.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import discord
+
+from bot.constants import Categories
+
+log = logging.getLogger(__name__)
+
+
+def is_help_channel(channel: discord.TextChannel) -> bool:
+    """Return True if `channel` is in one of the help categories (excluding dormant)."""
+    log.trace(f"Checking if #{channel} is a help channel.")
+    categories = (Categories.help_available, Categories.help_in_use)
```

Comment:
Why? I don't foresee it being used anywhere else, considering the helper functions are more convenient. It's not being used internally besides in this one function, so making it a "private" constant is also unjustified.

Reply:
I guess I'm used to static things being constants for readability, but it makes sense to keep it local if it's only going to get used here; not a huge deal so if you think this is more readable I believe the PR is ready then.

Reply:
You have a point, but I think that's situational. In this case, the only reason it's a variable is because it'd be too long/ugly to inline it. Furthermore, each category is already a constant on its own.